techniques/html/H39.html

```diff
@@ -27,16 +27,15 @@ <h3>An appointment calendar with a <code class="el">caption</code>
       </section>
    </section><section id="tests"><h2>Tests</h2>
       <section class="procedure"><h3>Procedure</h3>
-         <p>For each data table: </p>
+         <p>For each data table that has content that is presented as a title or identifier:</p>
```

Comment:
maybe instead of `indentifier`, more like `description`? or perhaps having that in addition to `identifier`? just because i've seen folks use `<caption>` to provide something more like a summary/explanation for a table.

Reply:
ah, expanding this PR further (to remind myself of the actual untouched content) I see that the "title or identifier" wording is actually in the main prose. while it goes beyond the scope of this PR, it *might* be worth tweaking it there too

Reply:
pushed commit to address. I was a little leery of using it since I didn't want to get into confusion over the old table summary attribute, but  I agree 'description' is better.